ansible/roles/common/tasks/fedora-install.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: set package manager
+  set_fact:
+    ansible_pkg_mgr: dnf
+  when: ansible_distribution_major_version|int >= 22
```

Comment:
This is a bug in ansible fixes in their development branch. Is this needed?  I'd rather not carry it...